cookbook/docs/contribute.rst

```diff
@@ -1,122 +1,321 @@
-   * `Integrations <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/integrations>`__: Contains examples that leverage one or more of the available plugins.
-   * `Case Studies <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/case_studies>`__: Contains examples that demonstrate the usage of Flyte to solve real-world problems. These are generally more complex examples that may require extra setup or that can only run on larger clusters.
-       
-2. Create a directory. (applicable for ``integrations`` and ``case_studies`` directories)
+   pyflyte run --remote my_example.py:wf --input hello
+
+Similarly, the ``run_my_example.py`` script should contain the python code that users need to invoke to run the script
+with ``FlyteRemote``, for example:
+
+.. code-block:: python
+   from flytekit.configuration import Config
+   from flytekit.remote import FlyteRemote
+
+   from my_example import wf
+
+   remote = FlyteRemote(
+       config=Config.for_endpoint("localhost:30081", insecure=True),
+       default_project="flytesnacks",
+       default_domain="development",
+   )
+
+   registered_workflow = remote.register_script(wf)
+
+   execution = remote.execute(registered_workflow, inputs={"input": "hello"})
+   print(f"Execution successfully started: {execution.id.name}")
+
+Finally, in the ``my_example.py`` example script, place the following custom sphinx directive:
+
+.. code-block::
+
+   .. run-example-cmds::
+
+This will insert a dropdown in the rendered documentation page with instructions on how to run the example
+you just created:
+
+.. image:: https://raw.githubusercontent.com/flyteorg/static-resources/main/flyte/contribution_guide/run_commands.png
+   :alt: Example run commands
+
+Test your code
+===============
 
-   After determining where to put your example, create a directory under the appropriate parent directory. Each example directory should contain:
+If the example code can be run locally, just use ``python <my file>.py`` to run it.
 
-   * Dockerfile
-   * Makefile
-   * README.rst
-   * __init__.py
-   * requirements.in
-   * sandbox.config
+Testing on a Cluster
+---------------------
 
-   It might be easier to copy one of the existing examples and modify it to your needs.
+Install Flytectl by running
 
-3. Add the example to CI.
+.. code-block::
+   brew install flyteorg/homebrew-tap/flytectl
 
-   Add the example to `flyte_tests_manifest.json <https://github.com/flyteorg/flytesnacks/tree/master/cookbook/flyte_tests_manifest.json>`__.
+.. note:: 
+   Learn more about installation and configuration of Flytectl `here <https://docs.flyte.org/projects/flytectl/en/latest/index.html>`__.
 
-4. Test your code!
+Run the ``flytectl sandbox start --source=$(pwd)`` command in the directory that's one level above the directory that
+contains workflows.
```

Comment:
I don't think we need to worry about the directory anymore cause `pyflyte run ...` should work irrespective of the directory in which the sandbox spins up.